src/scroll-timeline-css-parser.js

```diff
@@ -354,6 +376,50 @@ export class StyleParser {
     }
   }
 
+  extractScrollTimelineNames(contents) {
+    const value = RegexMatcher.ANIMATION_TIMELINE.exec(contents)[1].trim();
+    const timelineNames = [];
+
+    value.split(",").map(part => part.trim()).forEach(part => {
+      if(isAnonymousScrollTimeline(part)) {
+        const name = this.saveAnonymousTimelineName(part);
+        timelineNames.push(name);
+      } else {
+        timelineNames.push(part);
+      }
+    });
+
+    return timelineNames;
+  }
+
+  saveAnonymousTimelineName(part) {
+    // Anonymous scroll timelines are given a name that starts with ':' to
+    // prevent collision with named scroll timelines.
+    const name = `:t${this.nextAnonymousTimelineNameIndex++}`;
+    this.anonymousScrollTimelineOptions.set(name, this.parseAnonymousTimeline(part));
+    return name;
+  }
+
+  parseAnonymousTimeline(part) {
+    const openIndex = part.indexOf("(");
+    const closeIndex = part.indexOf(")");
```

Comment:
OK with this for now, but we should consider rewriting as this implementation is not strict. Off the top of my head, I cannot think of valid CSS that would be misinterpreted; however, we could erroneously flag invalid CSS as valid. For strictness, we should likely use a regex with capture groups.